DiamondLightSource/python-artemis
# 428 fix end writing to ispyb

Fixes #428

### To test:
1. Run new system test against code in main, confirm it fails
2. Run against this branch, confirm it passes
